bin/generate-commit-log

```diff
@@ -60,6 +60,8 @@ for message in messages:
         if re.search(pat, sub, re.I):
             should_skip = True
             break
+    if message.get("from") == "Renovate Bot <bot@renovateapp.com>":
+        should_skip = True
```

Comment:
This works great.  But--and it's nit-ish--let's move this check up to be the first thing in the loop, and `continue` when true.  That way we'll just skip subject regex matching.